packages/jest-cli/src/cli/index.js

```diff
@@ -130,9 +130,11 @@ const readResultsAndExit = (
   if (globalConfig.forceExit) {
     if (!globalConfig.detectOpenHandles) {
       console.error(
-        chalk.red(
-          'Force exiting Jest - have you considered using `--detectOpenHandles`?',
-        ),
+        chalk.red.bold('Force exiting Jest\n\n') +
+          chalk.red(
+            'Have you considered using `--detectOpenHandles`?\n\n to detect ' +
```

Comment:
Remove `?\n\n`? Seems like it's not necessary and looks, well, messy.

Reply:
Yeah, I noticed that